Makefile

```diff
@@ -216,6 +216,9 @@ deploy_staging: gcloud webapp_deps package_service var-BRANCH_NAME var-APP_PATH
 	rm -rf $(WPTD_PATH)revisions/service/wpt.fyi
 	rm -rf $(WPTD_PATH)api/spanner/service/wpt.fyi
 
+cleanup_staging_versions: gcloud
+	$(WPTD_GO_PATH)/util/cleanup-versions.sh
```

Comment:
This script executes `gcloud versions delete` - does that need a `-q` flag to avoid the requirement of user interaction?

Reply:
Indeed, I just pushed a change in `cleanup-versions.sh` to always delete stale versions without asking (by adding `--quiet`), as I think it should be pretty safe to do.